splatter.py

```diff
@@ -0,0 +1,32 @@
+
+import uuid
+from pymongo import MongoClient
+
+api_key_length = 30
+
+
+class SplatterService(object):
+
+    def __init__(self, database):
+        self.database = database
+
+    def create_api_key(self, name, key=None):
+
+        if key is not None and len(str(key)) != api_key_length:
+            raise ValueError("API key must be %s characters long")
```

Comment:
What do you mean by that? We should skip validation on explicitly included keys so I can have an API key that's the middle finger emoji if I want